app/send/page.tsx

```
@@ -174,7 +180,37 @@ export default function SendPage() {
   useEffect(() => {
     loadTransfers();
     loadContacts();
-  }, [loadTransfers, loadContacts]);
+  }, [loadTransfers, loadContacts, opts.token]);
+
+  const handleShowSendDialog = useCallback(() => setShowSendDialog(true), []);
```

ESLint

[error] 'handleShowSendDialog' is assigned a value but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'handleTabChange' is assigned a value but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] The value assigned to 'handleContactSelect' is not used in subsequent statements.

(no-useless-assignment)